### PR TITLE
Add Helm chart and Prometheus

### PR DESCRIPTION
**Pull request by m-abramovich**

### CODE REVIEW

File: helm/values.yaml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  # Database can be either external, like managed AWS RDS service, or internal - part of this chart.
+  # If it is internal, the dependency mentioned in Chart.yaml will be deployed
+  internal: true
+
+  # If external
+  db_name: kiwi
+  username: kiwi
+  # Set a password value via the command line argument:
+  # >>> helm upgrade --install --namespace <my_namespace> <helm_repo>/kiwi --set database.password=<my_password>
+  password:
```
[the remark this answers is not in the document]
Comment (m-abramovich) [reply helm/values.yaml]: fixed

File: tcms/settings/common.py [hunk 2]
```diff
@@ -171,6 +172,7 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "simple_history.middleware.HistoryRequestMiddleware",
+    "django_prometheus.middleware.PrometheusAfterMiddleware",
```
Comment (atodorov) [inline tcms/settings/common.py:175]: There are 2 independent items in this PR and IMO it's better to deal with them independently. 

Can you open a new PR which only contains the helm charts so I can review only that and work towards merging it.

**Review by atodorov — CHANGES_REQUESTED** [review]

File: tcms/settings/common.py (hunk 2 above)
Comment (m-abramovich) [reply tcms/settings/common.py:175]: @atodorov I will do, just need your help then. Helm chart has livenessProbe and readinessProbe that rely upon any URL that returns 200 OK code. This is why I have introduced metrics. Can you tell me please which other existing URL returns 200 so I could use it?

Comment (atodorov) [reply tcms/settings/common.py:175]: Any URL within Kiwi TCMS will return 200 when the container is up and running.  If you need a hard-coded 200 (e.g. can't deal with redirects) then `/accounts/login/` should work.

Comment (m-abramovich) [reply tcms/settings/common.py]: thank you
done

**Pull request merged**